### PR TITLE
Work on briefing pdf (DVZO-17)

### PR DESCRIPTION
**Pull request by haustre**
* Add i18n to jinja2
* Move briefing_pdf from pdf.py to dayplanning.py
* Use the samle dayplanning_context in html and pdf

### CODE REVIEW

**Review by busykoala — APPROVED** [review]
Merci, sieht gut aus! :smiling_face_with_three_hearts:

File: train_management/templates/latex/briefing.tex [hunk 1]
```diff
@@ -106,4 +107,62 @@
...
   {% endif %}
-  \end{document}
+
+  \begin{flushleft}
+  \textbf{Personal}
+  \end{flushleft}
+  \begin{table}[htbp]
+  {% set functions_table = namespace(row_count=0) %}
+    \begin{tabular}{|l|l|l|l|}
+      \hline Einsatzort & Funktion & Name & Telefon \\
```
Comment (osswald) [inline train_management/templates/latex/briefing.tex:117]: please use l10n

File: train_management/templates/latex/briefing.tex [hunk 2]
```diff
@@ -106,4 +107,62 @@
       \end{tabularx}
   \end{table}
   {% endif %}
-  \end{document}
+
+  \begin{flushleft}
+  \textbf{Personal}
```
Comment (osswald) [inline train_management/templates/latex/briefing.tex:112]: please use l10n

**Review by osswald — CHANGES_REQUESTED** [review]

**Comment by osswald** [comment]
will not be used

**Pull request closed without merging**